RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc

```diff
@@ -572,7 +572,7 @@ namespace {
 				       matched_pfcs.end()));	
       }
       for( const auto& clelem : best_comb ) {
-	const reco::PFClusterRef& clref = clelem->clusterRef();
+	//const reco::PFClusterRef& clref = clelem->clusterRef();
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant